.travis.yml

```diff
@@ -10,8 +10,6 @@ env:  # generate list with: $ tox -l | xargs -I ITEM echo "  - TOXENV="ITEM
   - TOXENV=py27-django14
```

Comment:
Right, I keep forgetting about that. I'd leave an issue open to fix that permanently, since it's a bit of a hassle to have to keep that up-to-date (all tests pass locally, BTW).

The configuration for this repo might serve as a reference:
 * [travis config](https://github.com/geier/khal/blob/master/.travis.yml)
 * [travis script](https://github.com/geier/khal/blob/master/.travis.run.sh)